apps/condo/bin/send-residents-add-account-request.js

```diff
@@ -0,0 +1,114 @@
+const conf = require('@core/config')
+
+const { BillingProperty } = require('@condo/domains/billing/utils/serverSchema')
+
+const { Message, Device } = require('@condo/domains/notification/utils/serverSchema')
+const {
+    RESIDENT_ADD_ACCOUNT_MANUAL_TYPE,
+    MESSAGE_SENT_STATUS, MESSAGE_DELIVERED_STATUS,
+    MESSAGE_READ_STATUS, MESSAGE_ERROR_STATUS,
+} = require('@condo/domains/notification/constants/constants')
+
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+const { Resident, ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
+
+const { BillingContextScriptCore, runIt, mapFieldUnique, mapToUsers } = require('./lib/billing-context-script-core')
+
+/**
+ This script sends push notifications to all users who are:
+     * residents of the properties:
+        - of the organization with provided billingContextId
+        - and have no accounts added
+     * have available pushTokens
+     * have not been sent this king of notifications during current month yet
+ */
+class ResidentsNotificationSender extends BillingContextScriptCore {
+    async proceed () {
+        const serviceConsumerWhere = { billingIntegrationContext: { id: this.billingContextId, deletedAt: null }, accountNumber_not: null, deletedAt: null }
+        const serviceConsumers = await this.loadListByChunks(ServiceConsumer, serviceConsumerWhere)
+        const consumerResidentIds = mapFieldUnique(serviceConsumers, 'resident.id')
+
+        console.info('[INFO] residents with account numbers found:', consumerResidentIds.length)
```

Comment:
you can use pino to make json logs

Reply:
It's manually run script. I'll add `pino` for automatic ones.